hq/markdown/vulnerability-management.md

```diff
@@ -1,14 +1,72 @@
 example of automated security at The Guardian is [Janus](https://github.com/guardian/janus). Unfortunately  there are some
 tasks that can't easily be automated, or we haven't got round to automating yet. These areas are discussed below.
 
+## Infrastructure vulnerabilities
+There are 5 tools to focus on in this section. They are in rough order of priority, so as a rule you should address Security HQ
+concerns before moving on to operating system patches. 
+
+### 1. Security HQ
+The first step of your long journey to the fountain of security involves a visit to [Security HQ](https://security-hq.gutools.co.uk).
+Here, you can review Security Groups, IAM credentials and S3 buckets for your account, and check to see if there are any
+problems. *Fix these first*.  
+
+Security HQ helpfully uses the colour RED to indicate stuff that we should probably be worried about. When it
+comes to S3 buckets, there may be a reason a bucket is publicly accessible, but there's a good chance that reason no longer
+applies.
+
+For security groups, you may have open access to a group if it is the group for a load balancer to a public service. 
+Otherwise, there should be some restrictions in place. There's no reason to allow open access to any EC2 instances - they
+should ideally be locked down to the load balancer and nothing else. Port 22 access should be removed everywhere.  
+
+With IAM credentials, note that any permanent credentials should be regularly rotated. A credential that is over 1,000
+days old, whilst no doubt full of wisdom, is a credential that needs rotating!
+
+### 2. Operating system patches
+You should be regularly replacing your AMIs to get the latest security patches. At present, our advice here is to use 
+[AMIable](amiable.gutools.co.uk/) to check the status of the AMIs used in your account. Most of our apps should be automatically
+on a recent AMI thanks to using a combination of riffraff scheduled deploys and riffraff [ami-cloudformation-parameter](https://riffraff.gutools.co.uk/docs/magenta-lib/types#amicloudformationparameter)
+deploy steps. If your app isn't deployed via riffraff then it might be more difficult to keep it up to date. Ideally there
+needs to be some kind of process, even if it's just a manual step when AMIable emails you about the out of date instances.
+
+### 3. AWS GuardDuty
+![guard gromit](./images/gromit.gif)
+
+If you're signed in to AWS already, you can click [here](https://eu-west-1.console.aws.amazon.com/guardduty/home?region=eu-west-1#/findings?macros=current)
+to take a look at AWS GuardDuty. You'll likely see a big list of activity going on in your AWS account that has been identified
+as concerning. The tool isn't infallible -
+for instance in the deploy tools account it flagged a load of instances as having unauthorised connections to the Tor
+network, but it turns out those instances are the ones we use for secure drop! So take it with some scepticism but 
```

Comment:
minor: SecureDrop is PascalCase

minor: Those images are used to perform healthchecks on hidden Tor sites. Not really worried about ambiguity in the docs but just stating for the record that we do not run SecureDrop on AMIs made by AMIgo. 😉 

It is possible to ignore findings is AWS GuardDuty if it is certain they are a false positive. For example, The InfoSec team have an EC2 instance that makes outbound requests via Tor to check the availability of hidden sites. This finding has been ignored if the instance talking to the Tor network matches certain tags. However, InfoSec would be very interested if an instance was attempting to contact the Tor network unexpectedly.